apps/vr-tests-react-components/src/utilities/TestWrapperDecorator.tsx

```diff
@@ -44,14 +44,6 @@ export const TestWrapperDecoratorFullWidth: DecoratorFunction<ExtendedStoryFnRet
 );
 
 const useNoAnimationStyles = makeStyles({
```

Comment:
since we now have only the paused name, could we call it this instead?
```suggestion
const usePausedAnimationStyles = makeStyles({
```